dev/com.ibm.ws.messaging.open_jms20_fat/build-test.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't see where this part is used.  If this part is now vestigial, consider removing it when the test suite is fully migrated.  The file serves only to confuse those who maintain the feature.